apps/ann-benchmarks/weaviate_import.py

```diff
@@ -36,22 +37,23 @@ def reset_schema(client: weaviate.WeaviateClient, efC, m, shards, distance):
 def load_records(client: weaviate.WeaviateClient, vectors, compression, dim_to_seg_ratio, override):
     collection = client.collections.get(class_name)
     i = 0
+    batch_size = 1000
+    len_objects = len(vectors)
     if vectors == None:
         vectors = [None] * 10_000_000
```

Comment:
might want to move the `len_objects = len(vectors)` down here. Otherwise it would be 0 in the case of vectors=None

Reply:
Thanks! I'll move it after the `if vectors == None`.